### DISCUSSION TITLE
Does it work on Raspberry clones?

### DISCUSSION THREAD

**Discussion by Piattica** [probonopd/MiniDexed#484] [Q&A] [answered]
Does it work on Raspberry clones?
Could you tell us which?

**Comment by diyelectromusic** [comment, marked as answer]
No, it relies on the circle "bare metal" environment which only works on genuine Raspberry Pi boards.  There is some discussion here: https://github.com/rsta2/circle/discussions/327 and more recently here: https://github.com/rsta2/circle/issues/358

There is MicroDexed for the Teensy though and it is possible to run Dexed itself on different Linux distributions (its available within Zynthian for example).

Kevin

**Comment by Piattica** [comment]
Thanks a lot for the fast answer
*       Paolo Piattica
 *


Il giorno gio 20 apr 2023 alle ore 17:48 Kevin ***@***.***>
ha scritto:
